### DISCUSSION TITLE
Disabling of Google Maps

### DISCUSSION THREAD

**Discussion by d1684023** [LiveHelperChat/livehelperchat#2009] [Q&A]
Hello.
How can I disable the Google Maps page after closing the chat? Because when we close the chat the page with Google Maps opens with the error of wrong API key. Yes, we don't have the API key and we don't need this function. Is it possible to return to the dashboard after closing a chat? I tried to disable GEO, but the redirect to the map is still there.
I was trying to search for it in preferences, in documentation, and in Google, but unfortunately, I couldn't find the answer. Could somebody please help? Thanks.

LiveHelper Chat version: 4.29

**Comment by remdex** [comment]
see chat configuration -> dashboard chat tabs. You can set any order you want
